numeral.js

```diff
@@ -249,6 +259,7 @@
             abbrB = false, // force abbreviation to billions
             abbrT = false, // force abbreviation to trillions
             abbrForce = false, // force abbreviation
+            omitAbbr = false,
```

Comment:
Yes. `abbrForce` allows you to specify exactly which abbreviation you want. So if you do `0,0aM` instead of `0,0a`, it'll format the number in millions, instead of defaulting to whatever it thinks is the best order to format by. 

In either case, `ommitAbbr` works the same. 

From the test cases: 
```
numeral(-5444333222111).format('0,0 aK~') === '-5,444,333,222'
numeral(-5444333222111).format('0,0 aM~') === '-5,444,333'
numeral(-5444333222111).format('0,0 aB~') === '-5,444'
numeral(-5444333222111).format('0,0 aT~') === '-5'
numeral(123456).format('0.0[0] aK~') === '123.46'
```